lib/carbonite/migrations.ex

```diff
@@ -126,6 +127,18 @@ defmodule Carbonite.Migrations do
     :ok
   end
 
+  defp initially_option(opts) do
+    opts
+    |> Keyword.get(:initially, "")
+    |> to_string()
+    |> String.upcase()
+    |> case do
+      "" -> "IMMEDIATE"
+      "IMMEDIATE" -> "IMMEDIATE"
+      "DEFERRED" -> "DEFERRED"
+    end
```

Comment:
```suggestion
    |> Keyword.get(:initially, :immediate)
    |> case do
      :immediate -> "IMMEDIATE"
      :deferred -> "DEFERRED"
    end
```

I'd either use atoms, or alternatively use the string directly as is in the query. But for me atoms look like a nicer API.

Reply:
Also just saw that I forgot to document that param here :+1: will go with atoms then